install/network/flow/install.yml

```diff
@@ -5,10 +5,17 @@ description: |
   Set up your network devices so they send network data to New Relic One.
 
 target:
-  type: integration
-  destination: cloud
+  type: agent
+  destination: host
+  os:
+    - linux
 
 install:
+  mode: nerdlet
+  destination:
+    nerdletId: network-performance-monitoring.setup-network-flow
```

Comment:
@kidk @jbeveland27 is calling this Nerdlet already working?

Reply:
ok, exactly we need to hold off merging this PR then
@nr-jbiggley can you work with the Justin and the Kentik product team who build that install nerdlet?

Reply:
Already in progress @polfliet

Reply:
bam!